modules/moa-accessing-your-cluster.adoc

```diff
@@ -20,16 +20,20 @@ For other supported IDPs, enter `moactl create idp --help`, and consult the Open
 [source, bash]
 ----
 $ moactl create idp --cluster rh-moa-test-cluster1 --type github                                                                                               
-
+----
++
+.Example output
+[source,terminal]
+----
 I: Loading cluster 'rh-moa-test-cluster1'                                                                                                                                                               
 I: Loading identity providers for cluster 'rh-moa-test-cluster1'                                                                                                                                        
 To use GitHub as an identity provider, you must first register the application:                                                                                                                 
 ? List of GitHub organizations or teams that will have access to this cluster: openshift-online                                                                                                 
```

Comment:
Also wanted to just call out the two Markdown links on lines 13 and 18. This may have been intended, but wanted to flag it if it should be changed to Asciidoc links.

Reply:
Good catch. I'll move these to related topics section in the assembly.